types/react-native/index.d.ts

```diff
@@ -6258,6 +6258,51 @@ export interface ShareStatic {
     dismissedAction: string
 }
 
+type AccessibilityChangeEventName = 'change' | 'announcementFinished';
+
+/**
+ * @see https://facebook.github.io/react-native/docs/accessibilityinfo.html
+ */
+export interface AccessibilityInfoStatic {
+    /**
+     * Query whether a screen reader is currently enabled. 
+     * Returns a promise which resolves to a boolean. The result is true when a screen reader is enabled and false otherwise.
+     */
+    fetch(): Promise<boolean>;
+    
+    /**
+     * Add an event handler. Supported events:
+     *  - change: Fires when the state of the screen reader changes. 
+     *            The argument to the event handler is a boolean. 
+     *            The boolean is true when a screen reader is enabled and false otherwise.
+     * 
+     * - announcementFinished: iOS-only event. Fires when the screen reader has finished making an announcement. 
+     *                         The argument to the event handler is a dictionary with these keys:
+     *                          - announcement: The string announced by the screen reader.
+     *                          - success: A boolean indicating whether the announcement was successfully made.
+     */
+    addEventListener(eventName: AccessibilityChangeEventName, handler: Function): void;
```

Comment:
iirc there’s a preference in the DT project to use function literals, e.g. `() => any`.

Reply:
Thanks!

Reply:
Oh man... I’m so sorry about not being more clear. I actually meant a function literal instead of using the class name `Function`. Can you take one more stab at it?

Reply:
Ah, that makes more sense.  Updated :).